src/test/java/org/itsallcode/openfasttrace/importer/TestSpecificationListBuilder.java

```diff
@@ -22,37 +24,150 @@
+        final SpecificationListBuilder itemsBuilder = createBasicListBuilder();
+        itemsBuilder.appendDescription(expectedDescription);
+        itemsBuilder.setTitle(expectedTitle);
+        final List<SpecificationItem> items = itemsBuilder.build();
+        assertThat(items.size(), equalTo(1));
+        assertThat(items.get(0).getId(), equalTo(ID));
+        assertThat(items.get(0).getDescription(), equalTo("description"));
+        assertThat(items.get(0).getTitle(), equalTo("title"));
+    }
+
+    private SpecificationListBuilder createBasicListBuilder()
+    {
+        final SpecificationListBuilder itemsBuilder = SpecificationListBuilder.create();
+        itemsBuilder.beginSpecificationItem();
+        itemsBuilder.setId(ID);
+        return itemsBuilder;
     }
 
     @Test
-    public void testDuplicateIdNotIgnored()
+    public void testBuildWithStatus()
     {
-        this.builder.beginSpecificationItem();
-        this.builder.setId(ITEM_ID);
-        this.builder.endSpecificationItem();
-        this.builder.beginSpecificationItem();
-        this.builder.setId(ITEM_ID);
-        this.builder.endSpecificationItem();
+        final SpecificationListBuilder itemsBuilder = createBasicListBuilder();
+        itemsBuilder.setStatus(ItemStatus.DRAFT);
+        final List<SpecificationItem> items = itemsBuilder.build();
+        assertThat(items.get(0).getStatus(), equalTo(ItemStatus.DRAFT));
+    }
 
-        assertThat(this.builder.getItemCount(), equalTo(2));
+    @Test
+    public void testBuildWithTags()
+    {
+        final SpecificationListBuilder itemsBuilder = createBasicListBuilder();
+        itemsBuilder.addTag("foo");
+        itemsBuilder.addTag("bar");
+        final List<SpecificationItem> items = itemsBuilder.build();
+        assertThat(items.get(0).getTags(), containsInAnyOrder("foo", "bar"));
     }
 
+    // [utest->dsn~ignoring-artifact-types-during-import~2]
+    @Test
+    public void testIgnoreArtifactOfType()
+    {
+        final SpecificationListBuilder ignoringBuilder = createListBuilderIgnoringImpl();
+        ignoringBuilder.beginSpecificationItem();
+        ignoringBuilder.setId(SpecificationItemId.createId("impl", "ignore", 1));
+        ignoringBuilder.endSpecificationItem();
+        ignoringBuilder.beginSpecificationItem();
+        final SpecificationItemId importedId = SpecificationItemId.createId("dsn", "import", 1);
+        ignoringBuilder.setId(importedId);
+        ignoringBuilder.endSpecificationItem();
+        final List<SpecificationItem> items = ignoringBuilder.build();
+        assertThat(items.size(), equalTo(1));
+        assertThat(items.get(0).getId(), equalTo(importedId));
+    }
+
+    private SpecificationListBuilder createListBuilderIgnoringImpl()
+    {
+        final SpecificationListBuilder ignoringBuilder = SpecificationListBuilder
+                .createIgnoringArtifactTypes(Arrays.asList("impl"));
+        return ignoringBuilder;
```

Comment:
Inline returned value.
Maybe add method parameter allowing to specify list of ignored artifact types